### PR TITLE
ARXIVOPS-1433: physics category descriptions

### PR DESCRIPTION
**Pull request by jweiskoff**
Adds category descriptions to many physics areas.

### CODE REVIEW

File: arxiv/taxonomy/definitions.py [hunk 1]
```diff
@@ -308,6 +308,12 @@
     'acc-phys': {
```
Comment (jimentwood) [inline arxiv/taxonomy/definitions.py:308]: should this be in physics.acc-phys and in archive:physics?

Comment (jweiskoff) [reply arxiv/taxonomy/definitions.py:308]: Yes. I'll back that out in a sec.

**Comment by jimentwood** [comment]
can math.MP be updated to be the same as math-ph?

 description': 'math.MP is an alias for math-ph. Articles in this category focus on areas of research '
                       'that illustrate the application of mathematics to 
                       'problems in physics, develop mathematical methods for '
                       'such applications, or provide mathematically rigorous '
                       'formulations of existing physical theories. Submissions '
                       'to math-ph should be of interest to both physically '
                       'oriented mathematicians and mathematically oriented '
                       'physicists; submissions which are primarily of interest '
                       'to theoretical physicists or to mathematicians should '
                       'probably be directed to the respective physics/math '
                       'categories'

**Review by jimentwood — APPROVED** [review]
looks good

**Review by mhl10 — APPROVED** [review]

**Pull request merged**